### ISSUE TITLE
Logging improvements

### ISSUE THREAD

**Issue by erinknight242** [HeadspringLabs/hsbot#102]
Understand how the logging works in Azure, so that we can look back to see what caused a crash. Log streaming works, but only if you have it running and can reproduce the issue. For phantom crashes, we are in the dark right now. It looks like we have logging turned on, but the existing logs are back from 2017, which may have been a previous configuration.

**Comment by erinknight242** [comment]
Yeah. Especially since the Log Streaming doesn't work across sessions... if you restart, you have to start a new streaming session so you miss anything that would have been logging during the restart.

**Comment by heebnerd** [comment]
Maybe ApplicationInsights would provide a more robust logging option...
